lib/matplotlib/rcsetup.py

```diff
@@ -589,22 +577,11 @@ def _validate_int_greaterequal0(s):
         raise RuntimeError(f'Value must be >=0; got {s}')
 
 
-def validate_hatch(s):
-    r"""
-    Validate a hatch pattern.
-    A hatch pattern string can have any sequence of the following
-    characters: ``\ / | - + * . x o O``.
-    """
-    if not isinstance(s, str):
-        raise ValueError("Hatch pattern must be a string")
-    _api.check_isinstance(str, hatch_pattern=s)
-    unknown = set(s) - {'\\', '/', '|', '-', '+', '*', '.', 'x', 'o', 'O'}
-    if unknown:
-        raise ValueError("Unknown hatch symbol(s): %s" % list(unknown))
-    return s
+def validate_hatch(hatch):
+    _validate_hatch_pattern(hatch)
```

Comment:
If we do just import this, I don't see much reason to define a method that just calls it... 

I guess these are "public", (and [documented](https://matplotlib.org/stable/api/rcsetup_api.html#matplotlib.rcsetup.validate_hatch)), so that could be a reason, but I would spell this as 

```python
validate_hatch = _validate_hatch_pattern
```